package.json

```diff
@@ -112,9 +113,11 @@
     "prewatch": "npm run clean",
     "compile": "npx shadow-cljs compile :extension",
     "release": "npx shadow-cljs release :extension",
+    "package": "npx vsce package --allow-star-activation",
     "vscode:prepublish": "npm run clean && npm run release"
   },
   "dependencies": {
+    "fast-glob": "^3.2.11",
```

Comment:
Glob-ing is tricky stuff. I don't know how to roll our own. If we can add some Clojure lib that does it, we can remove this one.

Also: Calva utilises the npm ecosystem a lot. Never brings us trouble beyond making npm audit happy. So I guess we have different experiences with JS deps. 😄

Reply:
It's a pity that the vscode globber stops working when there is no root folder. Seems that it is for no reason, because when there is one, it can glob the User scripts just fine.

Reply:
Oh right, the vscode globber is what I thought was built-in.